### PR TITLE
Features/notifications add recent chat

### PR DESCRIPTION
**Pull request by ds1sqe**
add recent chat id to notification and tested

# websocket data example
```
{
    "recent_chat_id": 183,
    "notifications": [
        {
            "channel_hashed_value": "415e0fce",
            "count": 69
        },
        {
            "channel_hashed_value": "aaasdgads",
            "count": 2
        }
    ]
}
```
# example 2
```
{
    "recent_chat_id": 182,
    "notifications": [
        {
            "channel_hashed_value": "415e0fce",
            "count": 68
        },
        {
            "channel_hashed_value": "aaasdgads",
            "count": 2
        }
    ]
}
```

### CODE REVIEW

**Comment by kreimben** [comment]
이건 도대체 왜 구현한거야?

File: notifications/signals.py [hunk 1]
```diff
@@ -18,15 +18,18 @@ def create_notifications(sender, **kwargs):
...
-    for noti in response:
-        noti: Notification
-        async_to_sync(channel_layer.group_send)(
-            f'{noti.receiver_id}',
-            {
-                'type': 'notifications.broadcast',
-                'user_id': noti.receiver_id
-            }
-        )
+    if type(chat) is Chat:
```
Comment (kreimben) [inline notifications/signals.py:21]: 이거 없애

**Review by ryankimjh00 — APPROVED** [review]
thanks

**Review by kreimben — APPROVED** [review]
approved.

**Pull request merged**